google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -655,6 +670,26 @@ public boolean isNull(int columnIndex) {
       return rowData.get(columnIndex) == null;
     }
 
+    @Override
+    public <T extends AbstractMessage> T getProtoMessageInternal(int columnIndex, T m) {
+      Preconditions.checkNotNull(
+          m, "Proto Message shouldn't be null, default instance from generated class can be used.");
+      try {
+        return (T)
+            m.toBuilder().mergeFrom(((ByteArray) rowData.get(columnIndex)).toByteArray()).build();
+      } catch (InvalidProtocolBufferException e) {
+        throw SpannerExceptionFactory.asSpannerException(e);
+      }
+    }
+
+    @Override
+    protected <T extends ProtocolMessageEnum> T getProtoEnumInternal(
+        int columnIndex, Function<Integer, ProtocolMessageEnum> method) {
+      Preconditions.checkNotNull(
+          method, "Lambda method shouldn't be null, use forNumber from autogenerated enum class.");
```

Comment:
just curious, why `Lambda method shouldn't be null`? 

If it is within the scope of this method only, can reword comment better.

Reply:
yeah, done.